modal/sandbox.py

```diff
@@ -332,12 +333,12 @@ async def wait(self, raise_on_termination: bool = True):
                     raise SandboxTerminatedError()
                 break
 
-    async def tunnels(self, timeout: int = 50) -> List[api_pb2.TunnelData]:
+    async def tunnels(self, timeout: int = 50) -> Dict[int, Tunnel]:
         """Get tunnel metadata for the sandbox.
 
         Raises `SandboxTimeoutError` if the tunnels are not available after the timeout.
 
-        Returns a list of `TunnelData` objects, which contain the tunnel metadata.
+        Returns a dictionary of `Tunnel` objects which are keyed by the container port.
```

Comment:
Can we also add a note about the API change (with specific version) here?

Reply:
yeah good idea